docs/tutorials/aws.md

```diff
@@ -414,6 +414,39 @@ You can configure Route53 to associate DNS records with healthchecks for automat
 
 Note: ExternalDNS does not support creating healthchecks, and assumes that `<health-check-id>` already exists.
 
+## Govcloud caveats
+
+Due to the special nature with how Route 53 runs in Govcloud, there are a few tweaks in the deployment settings.
+
+* An Environment variable with name of AWS_REGION set to either us-gov-west-1 or us-gov-east-1 is required.  Otherwise it tries to lookup a region that does not exist in govcloud and it errors out.
+
+```yaml
+env:
+- name: AWS_REGION
+  value: us-gov-west-1
+```
+
+* Route 53 in Govcloud does not allow aliases.  Therefore, container args must be set so that it uses CNAMES and a txt-prefix must be set to something.  Otherwise, it will try to create a TXT record with the same value than the CNAME itself, which is not allowed.
```

Comment:
```suggestion
* Route53 in Govcloud does not allow aliases. Therefore, container args must be set so that it uses CNAMES and a txt-prefix must be set to something. Otherwise, it will try to create a TXT record with the same value than the CNAME itself, which is not allowed.
```